redditInterface.py

```diff
@@ -0,0 +1,115 @@
+from fastapi import FastAPI
```

Comment:
so you're going to have the other parts of code interact with the DB through a rest API?

Reply:
That is the best way I could think of since we need them to interact. Alternatively, I can re format the code and make it into a class with the same methods.

Reply:
Since I don't what the DB is going to be composed of, I assumed this is going to be the structure. This later can be modified and improved upon and more methods can be added, such as requesting for specific tickers across the entire subreddit data.